components/Article/Embeds/YouTube/YouTubeEmbed.tsx

```diff
@@ -23,23 +23,50 @@ const YouTubeEmbed = ({
     return null
   }
 
-  const videoId = getYoutubeVideoId(url)
+  let opts = {}
+  let videoId = getYoutubeVideoId(url)
```

Comment:
Esse aqui pode ficar como const
```suggestion
  const videoId = getYoutubeVideoId(url)
```